### DISCUSSION TITLE
[x][closed] Daniel: I don't think the phenotype attribute should be required. [Study]

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#89] [Team Posts]
Daniel: Some studies such as Blueprint or ENCODE do not focus on a phenotype

**Comment by sveinugu** [comment]
@dzerbino Yes I did think about this. My solution in the example (which is from ENCODE) was to use the term_iri: "http://purl.obolibrary.org/obo/PATO_0000461", which is "Normal". We could provide this (or something) as a standard solution in the description. The advantage is that one could then also more simply search for only tracks that do not focus on a phenotype. But it might be a bit awkward.

**Comment by sveinugu** [comment]
@dzerbino @jmfernandez One new thought is that JSON schema contains a "default" keyword, which could be used to point to the "Normal" term. I am a bit uncertain how this works, though.

**Comment by dzerbino** [comment]
Default value sounds like a good compromise.

**Comment by jmfernandez** [comment]
I also agree with all of you, @sveinugu and @dzerbino . About the JSON Schema validator implementations, some of them ignore the "default" keyword, others acknowledge it when they are run in the correct mode, injecting the default values in the validated JSON file.

**Comment by sveinugu** [comment]
Move phenotype to sample
